src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanService.java

```diff
@@ -68,7 +68,9 @@ public ActionPlan createActionPlan(final ActionPlan actionPlan) {
         .with("action_plan_name", actionPlan.getName())
         .debug("Creating action plan");
 
-    ActionPlan savedActionPlan = saveActionPlan(actionPlan);
+    actionPlan.setActionPlanPK(null);
```

Comment:
a primary key of null feels very wrong

Reply:
`actionPlanPK` is set from the sequence in the database.
I think this `null` is set to make sure that we're not using a PK sent in the request